pagopa/pn-frontend-e2e
# Pn 9247 e 2 e mittente invio notifica digitale a destinatario con nessun domicilio di piattaforma nessun domicilio generale e con pec fornita dalla pa caso ok al primo tentativo

## Short description

Created new test: PA send a notification with "domicilio speciale" set

## List of changes proposed in this pull request

- Created new test: PA send a digital notification with "domicilio speciale" set, first try ok

## How to test

Simply `run mvn clean test -D"cucumber.filter.tags=@TA_invioNotificaConDomicilioSpecialeImpostato"` and verify if the test works fine.
